frontends/web/src/routes/account/send/send.tsx

```diff
@@ -651,27 +643,15 @@ class Send extends Component<Props, State> {
                 </div>
                 <Grid col="1">
                   <Column>
-                    <Input
-                      label={t('send.address.label')}
-                      placeholder={t('send.address.placeholder')}
-                      id="recipientAddress"
-                      error={addressError}
-                      onInput={this.handleFormChange}
-                      value={recipientAddress}
-                      className={hasCamera ? style.inputWithIcon : ''}
-                      labelSection={debug ? (
-                        <span id="sendToSelf" className={style.action} onClick={this.sendToSelf}>
-                        Send to self
-                        </span>
-                      ) : undefined}
-                      autoFocus>
-                      { hasCamera && (
-                        <button onClick={this.toggleScanQR} className={style.qrButton}>
-                          <img className="show-in-lightmode" src={qrcodeIconDark} />
-                          <img className="show-in-darkmode" src={qrcodeIconLight} />
-                        </button>
-                      )}
-                    </Input>
+                    <ReceiverAddressInput
+                      accountCode={this.getAccount()?.code}
+                      addressError={addressError}
+                      hasCamera={hasCamera}
+                      onClickScanQRButton={this.toggleScanQR}
+                      onClickSendToSelfButton={this.sendToSelf}
+                      onInputChange={this.handleFormChange}
```

Comment:
handleFormChange uses `id` of the event.target to update a send.tsx state. With this change the `id` is now hidden in the child. 

I believe it could set the state directly without using handleFormChange/event.target.id etc. wdyt?

https://github.com/digitalbitbox/bitbox-wallet-app/blob/c4375864155bffc353c11749e116d3adf2a0f9db/frontends/web/src/routes/account/send/send.tsx#L398

Reply:
If I see the `<ReceiveAddressInput />` correctly and the code block that you highlighted, when a user types in the receive address input (not clicking the send all checkmark) it'll modify the `recipientAddress` state.

>  believe it could set the state directly without using handleFormChange/event.target.id etc. wdyt?

Not 100% sure I understood this one. Did you mean we can make a new method in the parent that does `
```js
this.setState({recipientAddress},() => { 
   this.validateAndDisplayFee(true); 
 }));
``` 

in the parent (`send.tsx`) instead of using the general  handleFormChange? Yeah sure we can do that, but why if I may know? It's essentially the same thing.. maybe easier to read I guess (instead of guessing target.id we know it's `recipientAddress`)?

Reply:
yes, I think explicitly setting the state with
```js
this.setState({recipientAddress},() => { 
   this.validateAndDisplayFee(true); 
 }));
 ```
is much better than using `handleFormChange` and depending on DOM element id. Especially now that id is buried in `<ReceiverAddressInput` is more difficult to read and understand. I think in the long term we should try to remove `handleFormChange`